src/Components/ListenOn/ListenOn.js

```diff
@@ -19,28 +43,22 @@ export default function ListenOn() {
                     Subscribe On:
                 </h3>
 
-                <span className="listenOnChannel">
-                    <img className="logo" src={appleLogo} alt="apple logo" />
-                    APPLE PODCASTS
-                </span>
-                <span className="listenOnChannel">
-                    <img className="logo" src={googleLogo} alt="spotify logo" />
-                    GOOGLE PODCASTS
-                </span>
-                <span className="listenOnChannel">
-                    <img
-                        className="logo"
-                        src={spotifyLogo}
-                        alt="spotify logo"
-                    />
-                    SPOTIFY
-                </span>
+                {listenLinks}
             </div>
-            <span className="ListenOnbrowseAll"> BROWSE ALL {">"} </span>
-            <h3 className="listenOnDiscord">
+            <Link className="navLink" to="/About">
+                <span className="ListenOnBrowseAll"> BROWSE ALL {">"} </span>
+            </Link>
+
+            <div className="listenOnDiscord">
                 <img className="logo" src={discordIcon} alt="discord logo" />
-                Discord Server
-            </h3>
+                <a
+                    href="https://bit.ly/hoppedupdiscord"
```

Comment:
you could probably bring this link out and save it in a variable as well. Just because it could always change at some point and it might be easier to modify it in the future as a variable. But that's a nit pick